magento-research/pwa-studio
# Embed codesandbox examples

## Description

This PR embeds live codesandbox examples from the new [code-samples][] repository.
It also fixes some broken auto-generated links.

[code-samples]: https://github.com/magento-research/code-samples

## Related Issue

Closes #1339 

## Verification Steps

1. Navigate to the `pwa-devdocs` directory
2. Build and run the HTML preview server: `npm run start`
3. Navigate to: `/peregrine/reference/hooks/useSearchParam/`
4. Verify the CodeSandbox iframe is embedded on the page with the `useSearchParamExample.js` file open in the editor.
5. Click on 'Click to Run'
6. Verify the server runs and the editor's browser is pointing to: `/usesearchparam?query=hello+world`
7. Look for the line: "For implementation details View Source." and click on the 'View Source' link.
8. Verify the link is not broken.
9. Navigate to `/peregrine/reference/hooks/useQueryResult/`
10. Verify the CodeSandbox iframe is embedded on the page with the `useQueryResultExample.js` file open in the editor.
11. Click on 'Click to Run'
12. Verify the server runs and the editor's browser is pointing to: `/usequeryresult`

## Screenshots / Screen Captures (if appropriate)

## Checklist:
